src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -968,15 +968,15 @@ void func_8088E850(EnElf* this, PlayState* play) {
                 break;
 
             default:
-                arrowPointedActor = play->actorCtx.targetContext.arrowPointedActor;
+                fairyActor = play->actorCtx.targetCtx.fairyActor;
```

Comment:
```suggestion
                targetFairyActor = play->actorCtx.targetCtx.fairyActor;
```
I wonder if this should have `target` in front when separated from `targetCtx` to avoid thinking this is a pointer to the fairy actor and instead the actor that fairy is targeting